presto-docs/src/main/sphinx/release/release-330.rst

```diff
@@ -0,0 +1,131 @@
+=========================
+Release 330 (XX Feb 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect behavior of :func:`format` for ``char`` values. Previously, the function
+  did not preserve trailing whitespace of the value being formatted. (:issue:`2629`)
+* Fix query failure in some cases when aggregation uses inputs from both sides of a join. (:issue:`2560`)
+* Fix query failures when dynamic filtering is on for certain multi-level joins. (:issue:`2659`)
+* Fix failure of ``SELECT`` queries when accessing ``information_schema`` schema tables with empty
+  value used in predicate (like ``SELECT * FROM catalog.information_schema.tables WHERE table_name=''``).
+  (:issue:`2575`)
+* Fix query failure when :doc:`/sql/execute` is used with an expression containing a function call. (:issue:`2675`)
+* Fix query failure for certain co-located joins when dynamic filtering is enabled. (:issue:`2685`)
+* Fix failure in ``SHOW CATALOGS`` when the user does not have permissions to see any catalogs. (:issue:`2593`)
+* Improve query performance for some join queries when :doc:`/optimizer/cost-based-optimizations`
+  are turned on. (:issue:`2722`)
+* Prevent uneven distribution of data that could occur when writing data
+  if redistribution or writer scaling is enabled. (:issue:`2788`)
+* Add support for ``CREATE VIEW`` with comment (:issue:`2557`)
+* Add support for all major geometry types to :func:`ST_Points`. (:issue:`2535`)
```

Comment:
do we want to list those geometry types?

Reply:
It's a long list. I think it's overkill to list them all.